docs/src/main/sphinx/release/release-408.md

```diff
@@ -0,0 +1,71 @@
+# Release 408 (22 Feb 2023)
+
+## General
+
+* Fix query failure when performing a remote partitioning exchange on a
```

Comment:
"remote partitioning exchange" is not something normal users see or understand. Rephrase in terms of the kinds of queries that have this problem.